### PR TITLE
IfcPipeSegment Rail Import BIMQ 2021-07-21T06:39:53.0889322Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcPipeSegment

### CODE REVIEW

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
Psets need review see comments and are also missing documentation file

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcHvacDomain/PropertySets/Pset_PipeSegmentTypeCommon/DocPropertySet.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		<DocProperty xsi:nil="true" href="NominalDiameter_30E560qVCHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="InnerDiameter_37zO0qVCHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="OuterDiameter_3F7mG0qVCHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="Length_1tVAn5NDfEoeby73RRtvnw" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcHvacDomain/PropertySets/Pset_PipeSegmentTypeCommon/DocPropertySet.xml]: I modified this property to an existing Length property. Commit : fe24a74655fda523ae304aad215110433a2cff77

File: IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml:1]: Out of scope

File: IFC4x3/Properties/c/ComplementaryWorks_0NZNNBTeH4$RmON2XH6spL/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Describes the presence of “Transversal pipes for the removal of water”, “Wells lifting water” or “Dispersing wells”
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/c/ComplementaryWorks_0NZNNBTeH4$RmON2XH6spL/Documentation.md]: I have modified definition. Commit : fe24a74655fda523ae304aad215110433a2cff77

File: IFC4x3/Properties/c/ComplementaryWorks_0NZNNBTeH4$RmON2XH6spL/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ComplementaryWorks_0NZNNBTeH4RmON2XH6spL" Name="ComplementaryWorks" UniqueId="178d75cb-7684-44fd-bc18-5c28511b6cd5" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
+	<Enumeration xsi:type="DocPropertyEnumeration" xsi:nil="true" href="PEnum_ComplementaryWorks" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/c/ComplementaryWorks_0NZNNBTeH4$RmON2XH6spL/DocProperty.xml:3]: I have modified Enums and definition. Commit : fe24a74655fda523ae304aad215110433a2cff77

File: IFC4x3/Properties/c/CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Coating material
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/c/CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ/Documentation.md]: Done. Commit : fe24a74655fda523ae304aad215110433a2cff77

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PipeSegmentTypeDrainChannel" UniqueId="de04429f-827d-7f1b-ae7f-5c0b52a67542" ApplicableType="IfcPipeSegment/GUTTER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ" />
+		<DocProperty xsi:nil="true" href="ComplementaryWorks_0NZNNBTeH4RmON2XH6spL" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml:5]: I think the elements mentionned in the description are functions that could be done by DrainChannel. I modified the description and add the values to a PENUM. What do you think about it ? commit : fe24a74655fda523ae304aad215110433a2cff77

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+		<DocProperty xsi:nil="true" href="TributoryArea_2NfVFLXTvECOGJmSEJH8oI" />
+		<DocProperty xsi:nil="true" href="HeightAboveMeanSeaLevel_3pWIuI7GX20gqJiYrynJl" />
+		<DocProperty xsi:nil="true" href="HeightBelowSeaLevel_3TpCrbsffEa8AbkNFInavi" />
+		<DocProperty xsi:nil="true" href="GutterDesign_2ESBkrmA9FbQhweDWMv3LN" />
+		<DocProperty xsi:nil="true" href="GutterFunction_1ZXy2BRI1CGe5Q4qFeeTJ0" />
+		<DocProperty xsi:nil="true" href="IsCovered_2YzvRQEzP4dh97Y5815zNr" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml:11]: I think it depend on the situation! We have both models, fully assembled Drain Chanel or Pipe + cover.
So I think it's better to keep the property for the first case.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+		<DocProperty xsi:nil="true" href="IsAMonitoringFacility_3KGj0PEan4G86e5DwEt_YL" />
+		<DocProperty xsi:nil="true" href="TypeOfStructureSystemAssociated_2MIdt0r9v2SAGEe8CLCkcQ" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: Agree with you. I have erased it. Commit : 8c09c86ec82f06223521dccec4a23da686446357

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PipeSegmentTypeDrainChannel" UniqueId="de04429f-827d-7f1b-ae7f-5c0b52a67542" ApplicableType="IfcPipeSegment/GUTTER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ" />
+		<DocProperty xsi:nil="true" href="ComplementaryWorks_0NZNNBTeH4RmON2XH6spL" />
+		<DocProperty xsi:nil="true" href="TributoryArea_2NfVFLXTvECOGJmSEJH8oI" />
+		<DocProperty xsi:nil="true" href="HeightAboveMeanSeaLevel_3pWIuI7GX20gqJiYrynJl" />
+		<DocProperty xsi:nil="true" href="HeightBelowSeaLevel_3TpCrbsffEa8AbkNFInavi" />
+		<DocProperty xsi:nil="true" href="GutterDesign_2ESBkrmA9FbQhweDWMv3LN" />
+		<DocProperty xsi:nil="true" href="GutterFunction_1ZXy2BRI1CGe5Q4qFeeTJ0" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: Agree. I have erased the property! Commit : 8c09c86ec82f06223521dccec4a23da686446357

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 10]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PipeSegmentTypeDrainChannel" UniqueId="de04429f-827d-7f1b-ae7f-5c0b52a67542" ApplicableType="IfcPipeSegment/GUTTER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ" />
+		<DocProperty xsi:nil="true" href="ComplementaryWorks_0NZNNBTeH4RmON2XH6spL" />
+		<DocProperty xsi:nil="true" href="TributoryArea_2NfVFLXTvECOGJmSEJH8oI" />
+		<DocProperty xsi:nil="true" href="HeightAboveMeanSeaLevel_3pWIuI7GX20gqJiYrynJl" />
+		<DocProperty xsi:nil="true" href="HeightBelowSeaLevel_3TpCrbsffEa8AbkNFInavi" />
+		<DocProperty xsi:nil="true" href="GutterDesign_2ESBkrmA9FbQhweDWMv3LN" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: Agree. I have erased the property! Commit : 8c09c86ec82f06223521dccec4a23da686446357

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 11]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PipeSegmentTypeDrainChannel" UniqueId="de04429f-827d-7f1b-ae7f-5c0b52a67542" ApplicableType="IfcPipeSegment/GUTTER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ" />
+		<DocProperty xsi:nil="true" href="ComplementaryWorks_0NZNNBTeH4RmON2XH6spL" />
+		<DocProperty xsi:nil="true" href="TributoryArea_2NfVFLXTvECOGJmSEJH8oI" />
+		<DocProperty xsi:nil="true" href="HeightAboveMeanSeaLevel_3pWIuI7GX20gqJiYrynJl" />
+		<DocProperty xsi:nil="true" href="HeightBelowSeaLevel_3TpCrbsffEa8AbkNFInavi" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: Agree with you! Change to OrthometricHeight which is more generic! Commit : 8c09c86ec82f06223521dccec4a23da686446357

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+		<DocProperty xsi:nil="true" href="GutterDesign_2ESBkrmA9FbQhweDWMv3LN" />
+		<DocProperty xsi:nil="true" href="GutterFunction_1ZXy2BRI1CGe5Q4qFeeTJ0" />
+		<DocProperty xsi:nil="true" href="IsCovered_2YzvRQEzP4dh97Y5815zNr" />
+		<DocProperty xsi:nil="true" href="IsAMonitoringFacility_3KGj0PEan4G86e5DwEt_YL" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: I have changed the name. commit : 8c09c86ec82f06223521dccec4a23da686446357

[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: No informations on upperlevel systems. We'll keep it at this level.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 13]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PipeSegmentTypeDrainChannel" UniqueId="de04429f-827d-7f1b-ae7f-5c0b52a67542" ApplicableType="IfcPipeSegment/GUTTER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml:2]: ?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+	<Properties>
+		<DocProperty xsi:nil="true" href="CoatingMaterial_1gcAobXS9ASPgG_rbEIGRZ" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml:4]: I'm sorry, I didn't get the point?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml (hunk 13 above)
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: I have erased this pset and added properties to the Pset_PipeSegmentTypeGutter 523aca1cead9fea7474871cee96b141bf522686d

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml (hunk 7 above)
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: I add the property to ther pset_PipeSegmentTypeGutter  7e186197c5a3e89a7c3876d8911912590a08309d

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml (hunk 14 above)
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_PipeSegmentTypeDrainChannel/DocPropertySet.xml]: Agree! I have erased the property.

File: IFC4x3/Quantities/f/FootprintArea_0GG8jezen6axnfTFjEFf_m/DocQuantity.xml [hunk 15]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="FootprintArea" UniqueId="10408b68-f68c-4693-bc69-74fb4e3e9fb0" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Quantities/f/FootprintArea_0GG8jezen6axnfTFjEFf_m/DocQuantity.xml]: orphan

File: IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml [hunk 16]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK" Name="TRANSVERSAL_PIPES_FOR_REMOVAL_WATER" UniqueId="c43a3d88-6c19-4eb7-b1c5-6c382908ae54" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml]: Very Long suggest: TANSVERSAL_WATER_REMOVAL

Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml]: Documentation

File: IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml [hunk 17]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_ComplementaryWorks" Name="PEnum_ComplementaryWorks" UniqueId="ecd997a9-4827-4743-b223-8c7db9deab52">
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml:2]: Documentation

File: IFC4x3/PropertyConstants/l/LIFTING_WATER_WELLS_3cJBoQx6X2$RMoFjlvDYL_.xml [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LIFTING_WATER_WELLS_3cJBoQx6X2RMoFjlvDYL_" Name="LIFTING_WATER_WELLS" UniqueId="e64cbc9a-ec68-42fd-b5b2-3edbf936257e" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/l/LIFTING_WATER_WELLS_3cJBoQx6X2$RMoFjlvDYL_.xml]: Docuementation

File: IFC4x3/PropertyConstants/d/DISPERSING_WELLS_0c_7bXPd90nB0dUAVsQHIa.xml [hunk 19]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DISPERSING_WELLS_0c_7bXPd90nB0dUAVsQHIa" Name="DISPERSING_WELLS" UniqueId="26f87961-6672-40c4-b027-78a7f66914a4" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/d/DISPERSING_WELLS_0c_7bXPd90nB0dUAVsQHIa.xml]: Documentation

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
@fhulin Missing Documentation and some orphans once sorted LFTM!

File: IFC4x3/Quantities/f/FootprintArea_0GG8jezen6axnfTFjEFf_m/DocQuantity.xml (hunk 15 above)
Comment (fhulin) [reply IFC4x3/Quantities/f/FootprintArea_0GG8jezen6axnfTFjEFf_m/DocQuantity.xml]: Issue due to IfcDoc functionnality. So I have created a new quantity. Commit : 0bcb793ee763cd9139966920e98e2bff0489e4d5

File: IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml (hunk 16 above)
Comment (fhulin) [reply IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml]: Agree. 0bcb793ee763cd9139966920e98e2bff0489e4d5

Comment (fhulin) [reply IFC4x3/PropertyConstants/t/TRANSVERSAL_PIPES_FOR_REMOVAL_WATER_34EZs8R1bEjx75R3Wf2AvK.xml]: 0bcb793ee763cd9139966920e98e2bff0489e4d5

File: IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml (hunk 17 above)
Comment (fhulin) [reply IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml:2]: 0bcb793ee763cd9139966920e98e2bff0489e4d5

File: IFC4x3/PropertyConstants/d/DISPERSING_WELLS_0c_7bXPd90nB0dUAVsQHIa.xml (hunk 19 above)
Comment (fhulin) [reply IFC4x3/PropertyConstants/d/DISPERSING_WELLS_0c_7bXPd90nB0dUAVsQHIa.xml]: 0bcb793ee763cd9139966920e98e2bff0489e4d5

File: IFC4x3/PropertyConstants/l/LIFTING_WATER_WELLS_3cJBoQx6X2$RMoFjlvDYL_.xml (hunk 18 above)
Comment (fhulin) [reply IFC4x3/PropertyConstants/l/LIFTING_WATER_WELLS_3cJBoQx6X2$RMoFjlvDYL_.xml]: 0bcb793ee763cd9139966920e98e2bff0489e4d5

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM all conversations resolved

File: IFC4x3/Properties/i/IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL/DocProperty.xml [hunk 20]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL" Name="IsAMonitoredFacility" UniqueId="d442d019-3a4c-4440-81a8-14de8edfe895" PrimaryDataType="IfcBoolean" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL/DocProperty.xml]: Propose renaming to `IsMonitored`.

File: IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml [hunk 21]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Length" UniqueId="c388842f-4a93-46c9-8c00-a51f8c2a8fbe" PrimaryDataType="IfcPositiveLengthMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Length_33Y8GlIfD6oOm0fH_CAe__" Name="Length" UniqueId="c388842f-4a93-46c9-8c00-a51f8c2a8fbe" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml:2]: Reuse?

File: IFC4x3/Properties/o/OrthometricHeight_3pWIuI7GX20gqJ$iYrynJl/Documentation.md [hunk 22]
```diff
@@ -0,0 +1 @@
+The orthometric height is the vertical distance H along the plumb line from a point of interest to a reference surface known as the geoid, the vertical datum that approximates mean sea level
```
Comment (SergejMuhic) [inline IFC4x3/Properties/o/OrthometricHeight_3pWIuI7GX20gqJ$iYrynJl/Documentation.md]:
```suggestion
The orthometric height is the vertical distance H along the plumb line from a point of interest to a reference surface known as the geoid, the vertical datum that approximates mean sea level.
```

File: IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml [hunk 23]
```diff
@@ -0,0 +1,13 @@
...
+	<Documentation>Enumeration of the complementary function possible for the drain channel.</Documentation>
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="OTHER_0SAtxxJj15wQA15w2MsIV" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml]: Is the order important?

File: IFC4x3/Quantities/f/FootPrintArea_2$0cTYdvf6QO8z1nvVIBpB/DocQuantity.xml [hunk 24]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FootPrintArea_20cTYdvf6QO8z1nvVIBpB" Name="FootPrintArea" UniqueId="bf026762-9f9a-4669-823d-071e5f48bccb" />
```
Comment (SergejMuhic) [inline IFC4x3/Quantities/f/FootPrintArea_2$0cTYdvf6QO8z1nvVIBpB/DocQuantity.xml]: 1. Why not extract this from the model?
2. Is this a quantity?
3. No datatype set.

**Review by SergejMuhic — COMMENTED** [review]
Looks okay apart from the minor issues below.

File: IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml (hunk 21 above)
Comment (fhulin) [reply IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml:2]: I think, I'm already reusing a property.

Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml:2]: It is already a reuse yes

File: IFC4x3/Quantities/f/FootPrintArea_2$0cTYdvf6QO8z1nvVIBpB/DocQuantity.xml (hunk 24 above)
Comment (fhulin) [reply IFC4x3/Quantities/f/FootPrintArea_2$0cTYdvf6QO8z1nvVIBpB/DocQuantity.xml]: Right, it's a quantity! For the datatype, there is an issue on ifcdoc that didn't update the file... 
So I did it manualy. Commit : 48eb2f882610fb71dfd3c751fdc9e03736d59a7b

File: IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml (hunk 23 above)
Comment (fhulin) [reply IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml]: No, I changed it to have a more regular Order. 48eb2f882610fb71dfd3c751fdc9e03736d59a7b

File: IFC4x3/Properties/i/IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL/DocProperty.xml (hunk 20 above)
Comment (fhulin) [reply IFC4x3/Properties/i/IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL/DocProperty.xml]: Done. Commit : 48eb2f882610fb71dfd3c751fdc9e03736d59a7b

**Pull request merged**